app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapView.jsx

```diff
@@ -641,7 +638,14 @@ class SamplesHeatmapView extends React.Component {
 
   getControlOptions = () => ({
     // Server side options
-    metrics: this.props.metrics,
+    metrics: [
```

Comment:
Which should just filter `this.props.metrics` by value. It will guarantee that text is in sync with what is sent from server side. It will also make bugs less intrusive (?), if values change the option will simply no appear on the client side.